.ci/jenkins_job_build.sh

```diff
@@ -247,6 +247,13 @@ case "${CI_JOB}" in
 		export TEST_CRIO="true"
 	fi
 	;;
+"CRIO_K8S_MINIMAL")
```

Comment:
Nit: I dont know if could  help you but there is a function that you can call to set all env vars with the default values.
`init_ci_flags`. Just in case there are new in the future an you want to set vars explicitly.

Reply:
@jcvenegas, Do you mind if I don't change this now, but rather have a patch adding that as part of https://github.com/kata-containers/tests/pull/3071 ?

Reply:
sure